### DISCUSSION TITLE
LLM-based assistants for natural language explanations

### DISCUSSION THREAD

**Discussion by rolyp** [fluid-org/fluid-examples#21] [Ideas]
See for example [chatbot project](https://jceei.org/chatbot-project/) at University of Exeter, collaboration with Met Office which uses LLMs to produce natural language explanations based on Global Tipping Points report and Met Office UK Climate Projections.

I saw this presented at Climate Informatics 2024, it’s clearly a very important direction, but concerns about transparency are paramount. The researchers are explicitly interested in transparency, but what’s currently provided is pretty weak. (The responses say what section of the report they are derived from, but one could imagine much more fine-grained traceability.)

**Comment by RaoOfPhysics** [comment]
Is there an example of the chatbot in action? Can’t find it on that page. Looks very cool!
